litestar/_openapi/responses.py

```diff
@@ -240,14 +240,18 @@ def create_additional_responses(self) -> Iterator[tuple[str, OpenAPIResponse]]:
                 prefer_alias=False,
                 generate_examples=additional_response.generate_examples,
             )
-            schema = schema_creator.for_field_definition(
-                FieldDefinition.from_annotation(additional_response.data_container)
-            )
+            if additional_response.data_container is not None:
+                schema = schema_creator.for_field_definition(
+                    FieldDefinition.from_annotation(additional_response.data_container)
+                )
+                content = {additional_response.media_type: OpenAPIMediaType(schema=schema)}
+            else:
+                content = None
```

Comment:
Not sure if I should do this clause here or in `SchemaCreator.for_field_definition`?